src/ggml.c

```diff
@@ -3438,6 +3438,9 @@ inline static void ggml_vec_mad_f32(const int n, float * restrict y, const float
 
 //inline static void ggml_vec_scale_f32(const int n, float * y, const float   v) { for (int i = 0; i < n; ++i) y[i] *= v;          }
 inline static void ggml_vec_scale_f32(const int n, float * y, const float   v) {
+#if defined(GGML_USE_ACCELERATE)
+    vDSP_vsmul(y, 1, &v, y, 1, n);
+#endif
 #if defined(GGML_SIMD)
```

Comment:
```suggestion
#elif defined(GGML_SIMD)
```

Otherwise it will do the scaling twice

Reply:
oops, missed this